core/templates/dev/head/domain/exploration/EditableExplorationBackendApiService.js

```diff
@@ -113,8 +93,36 @@ oppia.factory('EditableExplorationBackendApiService', [
       });
     };
 
+    var _getExplorationUrl = function(explorationId, applyDraft, role) {
+      if (role) {
+        return UrlInterpolationService.interpolateUrl(
+        EDITABLE_EXPLORATION_DATA_ROLE_URL_TEMPLATE, {
```

Comment:
Is this spacing right? It seems like this would fail linting. I'd expect this and the next 3 lines to be indented rightward by 2, ditto for other similar ``return`` blocks.